python/tvm/relay/op/nn/nn.py

```diff
@@ -1993,17 +1993,27 @@ def batch_matmul(x, y):
     return _make.batch_matmul(x, y)
 
 
-def sparse_dense(data, weight):
+# pylint: disable=no-else-return,inconsistent-return-statements
+def sparse_dense(dense_mat, sparse_mat, sparse_lhs=False):
     r"""
-    Computes the matrix multiplication of `data` and `weight`, where `data` is
-    a dense matrix and `weight` is a sparse (either BSR or CSR) namedtuple with
+    Computes the matrix multiplication of `dense_mat` and `sparse_mat`, where `dense_mat` is
+    a dense matrix and `sparse_mat` is a sparse (either BSR or CSR) namedtuple with
     fields `data`, `indices`, and `indptr`.
 
     .. math::
 
-        \mbox{sparse_dense}(data, weight)[m, n] = \mbox{matmul}(x, \mbox{as_dense}(weight)^T)[m, n]
+       if sparse_lhs=True
```

Comment:
Is this going to render correctly inside of a math block?

Reply:
@tkonolige : You are right the rendering does not look good :)
With new change the new rendering looks as below:
![image](https://user-images.githubusercontent.com/32511895/101501597-f055d780-3995-11eb-989e-9bb71dd069fe.png)

I hope this is okay.